docs/docs/get_started/kubernetes_tutorial.md

```diff
@@ -1,867 +1,7 @@
 ---
 title: Kubernetes Tutorial
 id: kubernetes_tutorial
-layout: docs
 description: Secretless Broker Documentation
 permalink: /docs/get_started/kubernetes_tutorial.html
+redirect_to: https://docs.secretless.io/Latest/en/Content/Get%20Started/kube_tut_get_started.htm
```

Comment:
this link should now be https://docs.secretless.io/Latest/en/Content/Get%20Started/kubernetes_tutorial.htm